src/amo/components/Category/index.js

```diff
@@ -24,7 +18,10 @@ export class CategoryBase extends React.Component {
     category: PropTypes.object,
     dispatch: PropTypes.func.isRequired,
     errorHandler: PropTypes.object.isRequired,
+    filters: PropTypes.object,
     loading: PropTypes.boolean,
+    paginationQueryParams: PropTypes.object,
```

Comment:
Ah shoot, I didn't know that. Also I missed adding it, I'll send a quick follow-up now. Sorry 😓